lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -444,19 +465,49 @@ singleAddressFromKey _ xPub = Address $ BL.toStrict $ runPut $ do
 isolatePut :: Int -> Put -> Put
 isolatePut l x = do
     let bs = runPut x
-    if BL.length bs == (fromIntegral l)
+    if BL.length bs == convert l
     then putLazyByteString bs
     else fail $ "length was "
         ++ show (BL.length bs)
         ++ ", but expected to be "
         ++ (show l)
+  where
+    convert :: Int -> Int64
+    convert = fromIntegral
```

Comment:
```suggestion
    int64 :: a -> Int64
    int64 = fromIntegral
```

Reply:
I think it was fine with just `fromIntegral` and no helper here. Reverted.